controllers/cleaner_test.go

```diff
@@ -4,8 +4,8 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/onsi/gomega"
-	infrav1 "github.com/weaveworks/tf-controller/api/v1alpha2"
+	infrav1 "github.com/flux-iac/tofu-controller/api/v1alpha2"
+	"github.cgithub.com/flux-iac/tofu-controller
```

Comment:
Hi @akselleirv 
It seems the closing quote here is missing.

Reply:
Hi @chanwit,

Yes just saw that. Should be fixed in latest commit.